not-to-release/sources/email/enronsent33_01.xml.conllu

```diff
@@ -1198,8 +1198,8 @@
 
 # sent_id = email-enronsent33_01-0082
 # text = Robert C Williams@ENRON_DEVELOPMENT
-1	Robert	robert	X	GW	_	0	root	0:root	_
-2	C	c	X	GW	_	1	goeswith	1:goeswith	_
+1	Robert	robertc	X	GW	Typo=Yes	0	root	0:root	_
+2	C	_	X	GW	_	1	goeswith	1:goeswith	_
 3	Williams@ENRON_DEVELOPMENT	williams@enron_development	X	ADD	_	1	compound	1:compound	_
```

Comment:
Is this the correct tokenization? Not sure if it's supposed to be an email address or just an indication of name + org

Reply:
Oh looking at other examples I think token 3 should be `goeswith` rather than `compound`